### PR TITLE
ISOM 1152 table view for my pages

### PR DESCRIPTION
**Pull request by hanpuliu-charles**
## Problem

Closes [ISOM-1152](https://linear.app/ogp/issue/ISOM-1152/table-view-for-my-pages)
Component needed for dashboard view

## Solution

- Creating a draft of what the dashboard display may look like based on Figma mockups with dummy data. 
- Some changes will have to be made to call backend endpoint, fetch actual data and manipulated to fit expected type(Or change the code for the table to fit the data)

**Breaking Changes**

- [ ] Yes - this PR contains breaking changes
  - Details ...
- [X] No - this PR is backwards compatible

## Before & After Screenshots

**BEFORE**:

**AFTER**:

image

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -41,3 +41,6 @@ build-storybook.log
 
 # Build artifacts
 dist/
+
+# env file
+.env
```
Comment (harishv7) [inline .gitignore:46]: i think this exists at the root level .env right?

File: apps/studio/src/pages/dashboard/index.tsx [hunk 2]
```diff
@@ -1,17 +1,281 @@
...
-      <h1>DASHBOARD</h1>
+    <>
+      <TableContainer
+        w="100%"
+        borderRadius="8px"
+        border="1px"
+        borderColor="base.divider.medium"
+        bgColor="#FFF"
+      >
+        <Table>
```
Comment (harishv7) [inline apps/studio/src/pages/dashboard/index.tsx:138]: Usually we keep the stuff under the `pages` directory lean. It should mostly import components from the root components folder or the `/features` folder

Suggestion is to move this table layout under the `/features/dashboard` folder

File: apps/studio/src/pages/dashboard/index.tsx [hunk 3]
```diff
@@ -1,17 +1,281 @@
...
+                    <Td>
+                      {/* Fill doesn't seem to work with semantic tokens   */}
+                      {element.type === 'page' && element.status == 'draft' && (
+                        <Badge
+                          variant="subtle"
+                          colorScheme="warning"
+                          borderRadius="50px"
+                        >
+                          <BiSolidCircle
+                            fill="#FFDA68"
```
Comment (harishv7) [inline apps/studio/src/pages/dashboard/index.tsx:208]: why do we have a fill for the draft circle but not the published one?

Comment (hanpuliu-charles) [reply apps/studio/src/pages/dashboard/index.tsx:208]: The default color scheme of the badge applies the correct fill. Will put in the fill for the published circle for completeness sake!

File: apps/studio/src/pages/dashboard/index.tsx (hunk 2 above)
Comment (hanpuliu-charles) [reply apps/studio/src/pages/dashboard/index.tsx:138]: Understand the above, will shift the layout file to /features/dashboard

File: .gitignore (hunk 1 above)
Comment (hanpuliu-charles) [reply .gitignore:46]: Okay to remove, I think it was autogenerated by something

Comment (hanpuliu-charles) [reply .gitignore:46]: Hmm, don't quite get that, possible to clarify? Thanks!

Comment (harishv7) [reply .gitignore:46]: Thought this already exists, but saw it was missing.

**Review by harishv7 — APPROVED** [review]

File: apps/studio/src/pages/dashboard/index.tsx (hunk 3 above)
Comment (hanpuliu-charles) [reply apps/studio/src/pages/dashboard/index.tsx]: I have updated to wrap it in Chakra's icon component. Semantic tags now work and we no longer need the inline style tag.

**Pull request merged**